test/acceptance/test/profiles_test.go

```diff
@@ -129,6 +129,29 @@ Namespace: %s`, clusterName, namespace)))
 			resp, statusCode, err = kubernetesDoRequest(namespace, clusterName+"-"+profileName, "9898", "/healthz", clientSet)
 			return statusCode
 		}, "120s", "1s").Should(Equal(http.StatusOK))
+
+		By("Updating the version of the installed profile")
+		time.Sleep(4 * time.Minute)
```

Comment:
:thunk: is this the github API issue? Can we replace it with a robust `Eventually` instead?